src/PatientTestsApi/UpdatePatient/function.json

```diff
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
+        "get"
+      ],
+      "route": "patient/{registration-id}"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    }
+  ],
+  "scriptFile": "../dist/LoadPatient/index.js"
```

Comment:
This one should probably point to UpdatePatient/index.js. Definitely think you got the two function.json files mixed up.